sympy/series/fourier.py

```diff
@@ -154,9 +154,29 @@ def _eval_subs(self, old, new):
             return self
 
     def truncate(self, n=3):
-        """Returns the first n (non-zero)terms of the series.
+        """
+        Return the first n nonzero terms of the series.
+
+        If n is None return an iterator.
+
+        :param n: amount of non-zero terms in approximation
+        :type n: int or None
+        :rtype: Expr or iterator
+        :return: sigma approximation of function expanded into Fourier series
```

Comment:
Please change these to the numpydoc style as per convention in the sympy codebase.